### ISSUE TITLE
Vulnerability detector: same package being vulnerable and not vulnerable

### ISSUE THREAD

**Issue by cristgl** [wazuh/wazuh#1443]
## Bug report

**OS**
_CentOS7/RHEL7_

**Wazuh version**
_3.7.0_

**Install type**
_manager_

**Install method**
_sources_

*Log sample*

```
2018/09/25 08:55:49 wazuh-modulesd:vulnerability-detector: DEBUG: (5456): The 'kernel-tools' package from agent 5 is not vulnerable to CVE-2018-8897. Condition: package version (3.10.0-862.el7) less than 0:3.10.0-693.25.4.el7.
2018/09/25 08:55:49 wazuh-modulesd:vulnerability-detector: DEBUG: (5467): The 'kernel-tools' package from agent 5 is vulnerable to CVE-2018-8897. Condition: package version (3.10.0-862.el7) less than 0:3.10.0-862.2.3.el7.
2018/09/25 08:55:49 wazuh-modulesd:vulnerability-detector: DEBUG: (5456): The 'kernel-tools-libs' package from agent 5 is not vulnerable to CVE-2018-8897. Condition: package version (3.10.0-862.el7) less than 0:3.10.0-693.25.4.el7.
2018/09/25 08:55:49 wazuh-modulesd:vulnerability-detector: DEBUG: (5467): The 'kernel-tools-libs' package from agent 5 is vulnerable to CVE-2018-8897. Condition: package version (3.10.0-862.el7) less than 0:3.10.0-862.2.3.el7.
2018/09/25 08:55:49 wazuh-modulesd:vulnerability-detector: DEBUG: (5456): The 'python-perf' package from agent 5 is not vulnerable to CVE-2018-8897. Condition: package version (3.10.0-862.el7) less than 0:3.10.0-693.25.4.el7.
2018/09/25 08:55:49 wazuh-modulesd:vulnerability-detector: DEBUG: (5467): The 'python-perf' package from agent 5 is vulnerable to CVE-2018-8897. Condition: package version (3.10.0-862.el7) less than 0:3.10.0-862.2.3.el7.

```

Bug description
----------------
Two different conditions report vulnerable or not vulnerable the same package.

**Comment by crolopez** [comment]
This is due to the patch checking described [here](https://github.com/wazuh/wazuh/issues/1265).

**Comment by crolopez** [comment]
This was fixed in Wazuh 3.8.0.
